raddb/mods-config/sql/main/cassandra/queries.conf

```diff
@@ -49,7 +49,7 @@ sql_user_name = "%{User-Name}"
 # distrust the provided Event-Timestamp.
 #event_timestamp_epoch = "%l"
 
-event_timestamp_epoch = "%{%(integer:%{Event-Timestamp}):-%l}"
+event_timestamp_epoch = "%integer(%{&Event-Timestamp || %l})"
```

Comment:
That can now be

```
event_timestamp_epoch = %{((integer) &Event-Timestamp) || %l}
```

_cast_ to integer, as we don't need an `%integer()` function any more.  And the outer quotes aren't necessary.